src/main/java/core/basesyntax/countingoperations/CountOperationImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.countingoperations;
+
+import core.basesyntax.operationswithfile.Operation;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CountOperationImpl implements CountOperation {
+    @Override
+    public HashMap<String, Integer> getCount(HashMap<String, Integer> balance,
```

Comment:
In my opinion we should return Map if we want follow SOLID principles

Reply:
add